api/shapes/settings.ts

```diff
@@ -143,4 +149,13 @@ export const defaultSettings: Settings = {
   colorA11y: '-',
   wishListSource:
     'https://raw.githubusercontent.com/48klocs/dim-wish-list-sources/master/voltron.txt',
+  loStatSortOrder: [
+    '2996146975', //Mobility
+    '392767087', //Resilience
+    '1943323491', //Recovery
+    '1735777505', //Discipline
+    '144602215', //Intellect
+    '4244567218', //Strength
```

Comment:
Why not an array of numbers?

Reply:
There was something wanting me to do it in DIM, so it transferred over. Let me see if I can convert it over again in DIM when I get home this afternoon and I will update where with a change or a reason.

Reply:
Ah so I figured it out again, in DIM I am using this in LOSettings.tsx
```
<SortOrderEditor order={statOrder} onSortOrderChanged={statSortOrderChanged} />
```
the order parameter is a list of strings and so I turn the hashes into strings. This is so I can use the Stat manifest to get the localised version of the stats without us needed to have maintained translations.

The other option would be to change `SortProperty.id` type to be a `string | number`. That has some non-trivial flow on effects though, so I decided to just stringify the hash as it seemed easier and didn't appear to have any major drawbacks.

Reply:
It actually has an impact on how much space it takes in the database though - I think if we can make them numbers that'd be best.

Reply:
Yeah it will be doable so that is fine. I will sort out both PR's tonight.

Reply:
Should be good to go now